core/templates/dev/head/services/contextual/UrlServiceSpec.js

```diff
@@ -64,17 +64,17 @@ describe('Url Service', function() {
       var queryValue = '&value=1?';
       var queryField = 'field 1';
       var baseUrl = '/sample';
+      var encodedQueryField = encodeURIComponent(queryField);
+      var encodedQueryValue = encodeURIComponent(queryValue);
       expect(
         UrlService.addField(baseUrl, queryField, queryValue)).toBe(
-          baseUrl + '?' + encodeURIComponent(queryField) + '=' +
-          encodeURIComponent(queryValue)
+          baseUrl + '?' + encodedQueryField + '=' + encodedQueryValue
```

Comment:
No, I meant, literally having this be a hardcoded string. I.e. toBe('...') with no interpolations/additions.

Ditto below.

Reply:
Oh, ok, done.